tst/testrandom.g

```diff
@@ -1,34 +1,38 @@
-randomTest := function(collection, method, checkin...)
-    local test1, test2, test3, test4, test5, test6, localgen, checkmethod;
-    if Length(checkin) = 0 then
-        checkmethod := \in;
-    else
-        checkmethod := checkin[1];
-    fi;
+# Perform a variety of tests on Random generators.
```

Comment:
generators -> sources? methods? Not sure what is meant; perhaps a slightly longer description is needed?